### PR TITLE
Test UI create campaign

### PR DESCRIPTION
**Pull request by qxf2**
Added a test for creating a campaign. We are using the virtualized mailchimp service. This test will run only against localhost.

### CODE REVIEW

File: tests/integration/page_objects/form_object.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import conf.add_articles_locators_conf as locators
```
Comment (qxf2) [inline tests/integration/page_objects/form_object.py:1]: Can you please add this code to a create_newsletter_form_object.py?

File: tests/integration/tests/test_add_articles.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def test_add_article(test_obj):
         password = conf.password
         
         #Set the login
-        login = test_obj.login(email,password)
+        #login = test_obj.login(email,password)
```
Comment (qxf2) [inline tests/integration/tests/test_add_articles.py:34]: Don't comment out code. Delete it. We have git.

File: tests/integration/tests/test_create_campaign.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+def test_create_campaign(test_obj):
+
+    "Run the test"
+    #try:
+    if True:
+        #Initalize flags for tests summary
+        expected_pass = 0
+        actual_pass = -1
+        #Create a test object for edit an article
+        test_obj = PageFactory.get_page_object("create newsletter page")
```
Comment (qxf2) [inline tests/integration/tests/test_create_campaign.py:25]: Change this line to:

PageFactory.get_page_object("create newsletter page", base_url=test_obj.base_url)

File: tests/integration/tests/test_create_campaign.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        email = conf.email
+        password = conf.password
+        title = conf.title
+        category = conf.category
+        subject = conf.subject
+        opener = conf.opener
+        preview = conf.preview
+        
+        #Create Newsletter
+        result_create_newsletter = test_obj.create_newsletter(category,title)
```
Comment (qxf2) [inline tests/integration/tests/test_create_campaign.py:39]: Rename this to select articles

But still abstract all these into one `create_newsletter` method that selects articles, sets opener, etc.

File: tests/integration/tests/test_create_campaign.py [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        negative="The create newsletter preview is not filled")               
+
+        #click preview newsletter
+        click_preview_newsletter = test_obj.click_preview_newsletter()
+        test_obj.log_result(click_preview_newsletter,
+        positive="The preview newsletter is clicked",
+        negative="The preview newsletter is not clicked")                 
+
+        #create campaign
+        result_create_campaign = test_obj.create_campaign()
```
Comment (qxf2) [inline tests/integration/tests/test_create_campaign.py:69]: This needs a new page.

File: tests/integration/tests/test_create_campaign.py [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+        positive="The preview newsletter is clicked",
+        negative="The preview newsletter is not clicked")                 
+
+        #create campaign
+        result_create_campaign = test_obj.create_campaign()      
+        test_obj.log_result(result_create_campaign,
+        positive="Campaign is succesfully created",
+        negative="Campaign is not created")              
+
+    #except Exception as e:
```
Comment (qxf2) [inline tests/integration/tests/test_create_campaign.py:74]: Why has this been commented out?

File: tests/integration/tests/test_create_campaign.py [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+        #print("Exception when trying to run test: %s"%__file__)
+        #print("Python says:%s"%str(e))
+
+    assert expected_pass == actual_pass, "Test failed: %s"%__file__
+
+#---START OF SCRIPT
```
Comment (qxf2) [inline tests/integration/tests/test_create_campaign.py:80]: You can remove all lines from here.

**Review by qxf2 — COMMENTED** [review]
@ajitava-git please address the comments here.

**Comment by ajitava-git** [comment]
Merged and deleted

**Pull request merged**